ui/src/pages/Recovery/NewPassword.js

```diff
@@ -0,0 +1,78 @@
+import { Button, Input } from 'semantic-ui-react';
+import HyperLinkButton from '../../components/HyperLinkButton';
+import "../../App.css";
+import { useOutletContext } from 'react-router-dom';
+import ErrorLabel from '../../components/ErrorLabel';
+
+const NewPassword = () => {
+
+  const {
+    goBack,
+    checkPasswords,
+    onChangePassword,
+    onChangeConfirmPassword,
+    MIN_PASSWORD_LENGTH,
+    isValidPassword,
+    passwordsMatch,
+    password,
+    showError
+  } = useOutletContext();
+
+
+  return (
+    <>
+      <div className='headercontainer'>
+        <h1 className='mainheadline'>Create a password</h1>
+        <p className='subtext'>Your password will unlock the Manta Signer.</p>
+      </div>
+      <br />
+      <br />
+      <div>
+        <Input
+          className='input ui password'
+          type="password"
+          placeholder="Password (8 characters min)"
+          onChange={(e) => onChangePassword(e)}
+        />
+      </div>
+      <div>
+        <Input
+          className='input ui password'
+          type="password"
+          placeholder="Confirm Password"
+          onChange={(e) => onChangeConfirmPassword(e)}
+        />
+      </div>
+      {!isValidPassword && password.length > 0 && showError ?
+        <>
+          <br />
+          <ErrorLabel
+            text={"Please enter a minimum of " + MIN_PASSWORD_LENGTH + " characters."}
+          />
+        </>
+        :
+        (
+          !passwordsMatch && showError ?
+            <>
+              <br />
+              <ErrorLabel
+                text={"Passwords do not match."}
+              />
+            </>
+            :
+            <><br /><br /><br /></>
+        )}
```

Comment:
It's usually best to avoid using `<br>` for spacing, as opposed to css

Reply:
tailwind.css is really good for this, might want to consider setting that up. If you want e.g. 3rem of margin under some component, you can just add `mb-3` to its `className`. Optional, only do this if you find it useful.

Reply:
I removed all the <br>s, I think adding tailwind would be a great idea, but due to time constraints I don't think it will be possible right now, I added a todo in the css file in any case.